src/components/Expandable/Expandable.stories.tsx

```diff
@@ -1,6 +1,6 @@
-import { expect } from '@storybook/jest';
+import { expect } from '@storybook/test';
```

Comment:
`@storybook/jest` has been removed and replaced by `@storybook/test`, so went through and changed these after the calls failed.